sdk/loadtestservice/load-testing-rest/README.md

```diff
@@ -0,0 +1,192 @@
+# Azure Developer LoadTesting client library for JavaScript
```

Comment:
Azure Load Testing client library for JavaScript

Reply:
Updated.